### ISSUE TITLE
Add "Yes/no" language to ✔ - marks in the system status

### ISSUE THREAD

**Issue by helgatheviking** [woocommerce/woocommerce#26718]
**Is your feature request related to a problem? Please describe.**

The support emails I receive include the system status. But I'm finding it challenging to parse a ✔ out to determine (For example) if the site is connected to WooCommerce.com

**Describe the solution you'd like**
Marks should include text as well, ie : Yes ✔ and No ❌

**Comment by juliaamosova** [comment]
Hi @helgatheviking,

Thanks for this suggestion. We chatted about this briefly today with the team and agreed that it will be a good addition to the System Status Report. 

Please feel free to submit a PR if you like. Otherwise, I can work on it sometime this week.

**Comment by helgatheviking** [comment]
I'm a little overwhelmed at the moment. And I did figure out how to parse the checkmark so I'm probably not going to jump into this one. 

Totally unrelated, but could you take a look at https://github.com/woocommerce/woocommerce/issues/26584 ? I'd find the time to work on that some more if needed (already sent a PR) as that is going to enable a very cool feature in Name Your Price if it's included.

**Comment by juliaamosova** [comment]
>I'm a little overwhelmed at the moment. And I did figure out how to parse the checkmark so I'm probably not going to jump into this one.

@helgatheviking sounds good! 

>Totally unrelated, but could you take a look at #26584 ? I'd find the time to work on that some more if needed (already sent a PR) as that is going to enable a very cool feature in Name Your Price if it's included.

Sorry for the delay on this one! I see that you already submitted PR so I asked our PR reviewer for this week to make sure to prioritize it. You should receive a feedback from us on the PR between today and tomorrow 🙌

Thanks for your patience!

**Comment by jgreys** [comment]
Hi everyone! Just wondering, is it still relevant? :)

**Comment by helgatheviking** [comment]
Not really. I took a stab at it the other day, and got through all the yes items. But then I realized yes and no would be translatable. so for the purpose where I'm parsing the system status via a webhook when I get a support ticket, I wouldn't be able to match "yes" and "no" consistently anyway. and did eventually figure out how to work around that. So I don't need this any more myself.

**Comment by mfs-mindsize** [comment]
@juliaamosova - I picked this out of the "good first issue" list. 

As @helgatheviking mentioned, this solution would need language / localization. Unfortunately, I've been running into problems with the npm run build, as mentioned here:

https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment#configure-a-local-wordpress-instance

What would be the best way for me to get some additional support? The "try this" isn't helpful, at least to me. Which memory_limit, it doesn't seem to be my "main" PHP. It looks like this WP build is running its own. If you can help me, I can try to tackle this ticket.

**Comment by helgatheviking** [comment]
@mfs-mindsize personally I prefer localwp.com for setting up a local test instance. you do still need git/composer/nodeJS/npm on your local computer. and ideally a decent terminal.

**Comment by mfs-mindsize** [comment]
@helgatheviking - Thanks. I do have all those things (i.e., Git, Composer, etc.) Everything else I work on is functioning as expected. The WC assets build is my only known problem :( 

The issue seems to be WC is using its own version of PHP during the build, but there's no instruction on where that version keeps its memory_limit setting. I say this because I've made changes to my default (php.ini) and despite increasing the limit to 2+GB, I'm still getting the build error. 

Certainly, if this error is so common that it's in the docs, the docs could be a bit more helpful for the troubleshooting. 

What am I missing?

**Comment by helgatheviking** [comment]
You aren't by chance having this error are you #27984 b/c I just remembered that I've had issues with the dev build scripts here too... but then I feel like it just isn't a day at work if I'm not fighting a build script somewhere (largely because I'm on a Windows machine). I definitely agree the build/dev docu could use some more attention.

**Comment by mfs-mindsize** [comment]
@helgatheviking - This is the specific section of the docs I've been referring to:

https://share.getcloudapp.com/d5uPEzDQ

I use Windows as well :)  But in this case, I'm using my MBP for work.

**Comment by mfs-mindsize** [comment]
@helgatheviking - The solution to my memory_limit issue was to make it -1, which seeks a bit extreme but it works so for now I'll settle for that. 

I'm going to take a stab at this ticket. You mentioned above you've looked at it. Is there anything you'd like to share with me?  The html-admin-page-status-report.php seems straightforward. But there also appears to be a version of the report where JS gets involved. The latter might be less straightforward. 

For the sake of progress, I might split it in two. Your thoughts?

**Comment by mfs-mindsize** [comment]
Here's an example (of what I'm thinking) for the table row: WooCommerce REST API package

As you can see, I tried to address what looked to be accessibility gaps. I'm also questioning whether 'yes' and 'no' is the ideal verbiage. I've no dog in the verbiage race per se, just trying to open a (brief) conversation. Hopefully :)

Once I can get some feedback on the code below I can refine and then apply that down the page.  Thanks.

```
<?php
$version = wc()->api->get_rest_api_package_version();

if ( ! is_null( $version ) ) {
	echo '<mark class="yes"><span class="dashicons dashicons-yes" aria-hidden="true"></span>';
	echo '<span class="screen-reader-text">' . esc_html__( 'Status is good:', 'woocommerce' ) . '</span>';
	echo '<span class="yes-text"><span> ' . esc_html__( 'Good', 'woocommerce' ) . '</span></span> - ';
	echo '<span class="screen-reader-text">' . esc_html__( 'Version', 'woocommerce' ) . '</span>' . esc_html( $version );
	echo ' <code class="private">' . esc_html( wc()->api->get_rest_api_package_path() ) . '</code></mark> ';
} else {
	echo '<mark class="error"><span class="dashicons dashicons-warning" aria-hidden="true"></span>';
	echo '<span class="screen-reader-text">' . esc_html__( 'There\'s an error:', 'woocommerce' ) . '</span>';
	echo '<span class="no-text"><span> ' . esc_html__( 'Error', 'woocommerce' ) . '</span></span> - ';
	echo esc_html__( 'Unable to detect the REST API package.', 'woocommerce' ) . '</mark> ';
}
?>
```

**Comment by rrennick** [comment]
@mfs-mindsize It would be better to stick to the Yes/No verbiage. It's a status report vs an assessment of issues with the site. If, for example, a site has a theme doesn't register WooCommerce support, it could be an issue or it could be that the site has a custom theme that works as designed.

**Comment by mfs-mindsize** [comment]
@rrennick - Will do. I'll make the necessary adjustments. Thx. 

Anything else?

**Comment by rrennick** [comment]
@mfs-mindsize Did you create a PR for this? If so, it's not linked to this issue. The `Closes #{issue-id}` in the PR template adds the link.

**Comment by mfs-mindsize** [comment]
@rrennick - PR submitted: https://github.com/woocommerce/woocommerce/pull/28935

**Comment by nielslange** [comment]
Closing this issue due to https://github.com/woocommerce/woocommerce/pull/28935 and https://github.com/woocommerce/woocommerce/pull/37524.
